lib/spree/core/controller_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Spree
+  module Core
+    module ControllerHelpers
+      def self.included(klass)
+        klass.class_eval do
+          include Spree::Core::ControllerHelpers::Common
+          include Spree::Core::ControllerHelpers::Auth
+          include Spree::Core::ControllerHelpers::RespondWith
+          include Spree::Core::ControllerHelpers::Order
+        end
+      end
```

Comment:
I wonder if this not the same as 

```ruby
module ControllerHelpers
include Spree::Core::ControllerHelpers::Common
  include Auth
  include RespondWith
  include Order
end
```

it seems to me.

Reply:
yes! I thought about it, there is class loading and order of loading. I was not 100% sure it wouldnt cause trouble so I left it as is. There's plenty to improve, I tried to keep to the very simple improvements to reduce risk.